wagtail/wagtailadmin/views/chooser.py

```diff
@@ -22,25 +22,29 @@ def get_querystring(request):
 def browse(request, parent_page_id=None):
     ITEMS_PER_PAGE = 25
 
-    page_type = request.GET.get('page_type') or 'wagtailcore.page'
-    content_type_app_name, content_type_model_name = page_type.split('.')
+    page_types = request.GET.get('page_types', 'wagtailcore.page').split(',')
 
-    try:
-        content_type = ContentType.objects.get_by_natural_key(content_type_app_name, content_type_model_name)
-    except ContentType.DoesNotExist:
-        raise Http404
-    desired_class = content_type.model_class()
+    desired_classes = []
+    for page_type in page_types:
+        content_type_app_name, content_type_model_name = page_type.split('.')
```

Comment:
This will also allow you to delete the next few lines, and completely avoid doing a database lookup for the ContentType.